apps/cowswap-frontend/src/modules/tradeFormValidation/hooks/useTradeFormValidationContext.ts

```diff
@@ -34,6 +35,8 @@ export function useTradeFormValidationContext(): TradeFormValidationCommonContex
 
   const isPermitSupported = useTokenSupportsPermit(inputCurrency, tradeType)
 
+  const isZeroBalanceOrderAllowed = tradeType === TradeType.LIMIT_ORDER
```

Comment:
should we allow also SWAP Gnosis Safe?
I think it would help DAOs

Reply:
Although, maybe they can just place a LIMIT ORDER :)

Reply:
I would have it only for limit for now.